plugin/plugins.qmltypes

```diff
@@ -4,14 +4,14 @@ import QtQuick.tooling 1.2
 // It is used for QML tooling purposes only.
 //
 // This file was auto-generated by:
-// 'qmlplugindump -nonrelocatable @@ModulePrefix@@Connman 0.2'
+// 'qmlplugindump -nonrelocatable MeeGo.Connman 0.2'
```

Comment:
This was here on purpose and it has nothing to do with "remove unused var" which is the commit that now changes it

Reply:
fixed